### PR TITLE
fix: fixing popover breaking combobox focusing

### PR DESCRIPTION
**Pull request by bclark-p44**
DES-352

## 📝 Description

> Add a brief description

## Screenshots

> Please provide screenshots for any visual changes

## Merge checklist

- [ ] Added/updated tests
- [ ] Added changeset

### CODE REVIEW

**Comment by bclark-p44** [comment]
Still need to write tests

File: packages/react/src/components/DatePicker/DatePicker.tsx [hunk 1]
```diff
@@ -147,17 +147,18 @@ export const DatePicker = createComponent<'div', DatePickerProps>((props, forwar
...
-          isOpen={state.isOpen}
-          onClose={() => state.setOpen(false)}
-          overlayProps={dialogProps}
-          overlayRef={popoverRef}
-          placement="bottom start"
-          triggerRef={triggerRef}
-        >
-          <Calendar className="manifest-datepicker__calendar" {...calendarProps} />
-        </Popover>
+        {state.isOpen && (
```
Comment (Ashley-p44) [inline packages/react/src/components/DatePicker/DatePicker.tsx:150]: why the move from prop to here?

**Review by Ashley-p44 — APPROVED** [review]
Looks good, just had one question.

File: packages/react/src/components/DatePicker/DatePicker.tsx (hunk 1 above)
Comment (bclark-p44) [reply packages/react/src/components/DatePicker/DatePicker.tsx:150]: No good reason tbh, will move it back

**Pull request closed without merging**